app/src/androidTest/java/mozilla/lockbox/uiTests/Navigator.kt

```diff
@@ -153,11 +152,9 @@ class Navigator {
         if (goManually) {
             bypassOnboarding()
         } else {
+            // Go to ItemList after unlocking the device
+            Dispatcher.shared.dispatch(RouteAction.LockScreen)
             Dispatcher.shared.dispatch(LifecycleAction.UseTestData)
-            log.info("blocking for the routes")
-            // block until onboarding
-            // tap skip
-            blockUntil(RouteStore.shared.routes, RouteAction.ItemList)
```

Comment:
Was this the problem?

Reply:
Yes, the app is not able to go now from Welcome screen to  Item list automatically as it used  to do before that commit that broke the tests.

Reply:
@jhugman you can see that issue by running one of the tests/test suites. The app is stuck in the Welcome screen, for example run `AppRoutePresenterTest.kt`